x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_exploration/components/regression_exploration/evaluate_panel.tsx

```diff
@@ -65,6 +65,7 @@ export const EvaluatePanel: FC<Props> = ({ jobConfig, jobStatus, searchQuery })
   const {
     services: { docLinks },
   } = useMlKibana();
+  const mlApiServices = useMlApiContext();
```

Comment:
I believe `mlApiServices` is included in `services` provided by `useMlKibana` as part of `mlServices` so could do something like:

```
 const {
    services: { docLinks, mlServices: { mlApiServices } },
  } = useMlKibana();
  ```

Reply:
Yes, it's part of the kibana context, although it's quite deeply nested and with the prettier formatting it quite often ends up as multiple lines using the destructuring. I'd consider the hook a "shortcut" of sorts. On `main` there's also quite some similar usage where we use `useMlKibana()` to get some general services and more specific hooks like `useMlApiContext()` or `useToastNoticiations()` in the same place so I'd prefer to leave this as is.